### PR TITLE
Prepare release 0.10.0beta3

### PR DESCRIPTION
**Pull request by notoraptor**
@nouiz

### CODE REVIEW

File: NEWS.txt [hunk 1]
```diff
@@ -3,58 +3,40 @@ Release Notes
...
 We recommend that every developer updates to this version.
 
 Highlights:
- - Support NumPy ``1.13``
- - Support pygpu ``0.7``
- - Added conda recipe
- - Optional faster optimization step with new destroy handler
- - Added documentation for RNNBlock
- - Bug fixes, crash fixes, warning improvements and documentation updates
+ - conda packages for releases now available and updated in our own conda channel ``mila-udem``
```
Comment (nouiz) [inline NEWS.txt]: Add something like:

to use them for the beta packages, use this command "conda install -c mila-udem/label/pre theano pygpu"

**Comment by notoraptor** [comment]
Updated to make it clearer.

**Comment by nouiz** [comment]
jenkins test this

**Comment by nouiz** [comment]
Old jenkins run that failed before the restart: http://earlgrey.iro.umontreal.ca:8080/job/Theano_PR/859/

**Comment by notoraptor** [comment]
Updated to skip a MPI test (instead of just return) when it cannot run.

Jenkins still fails for that MPI test, and I don't know why as this test passes on my computer (after installing openmpi and mpi4py).

http://earlgrey.iro.umontreal.ca:8080/job/Theano_PR/862/

@nouiz

**Pull request merged**